liiweb/templates/liiweb/locality_legislation_list.html

```diff
@@ -17,6 +17,11 @@
   <h1 class="my-4">{{ page_heading }}</h1>
   {% include 'lawlibrary/_legislation_alert.html' %}
 {% endblock %}
+{% block entity-profile %}
```

Comment:
This is not necessary because legislation_list.html already handles the profile.